pkg/dependencies/factory.go

```diff
@@ -1371,6 +1394,62 @@ func (f *Factory) WithPackageControllerClient(spec *cluster.Spec, kubeConfig str
 	return f
 }
 
+// WithPackageControllerClientWithoutWait builds a package controller client that doesn't wait for active bundles.
+func (f *Factory) WithPackageControllerClientWithoutWait(spec *cluster.Spec, kubeConfig string) *Factory {
```

Comment:
Can we just add a variadic argument to the original constructor method?

Maybe that won't work, but we should avoid having all this logic copy/paste if we can